src/bs_translator_backend/services/document_conversion_service.py

```
@@ -182,7 +181,7 @@ async def convert_to_docling(
             if content_type is None:
                 content_type: str = get_mimetype(Path(filename))
 
-        assert isinstance(content_type, str)
+        assert isinstance(content_type, str)  # noqa: S101
```

Ruff (0.14.10)

Unused `noqa` directive (non-enabled: `S101`)

Remove unused `noqa` directive

(RUF100)